src/media/js/update_banner.js

```diff
@@ -0,0 +1,47 @@
+define('update_banner',
+    ['apps', 'core/capabilities', 'jquery', 'core/l10n', 'core/log', 'core/nunjucks', 'core/settings', 'core/utils', 'core/z'],
```

Comment:
Wrap 80 chars here and some of the lines below. And painfully alphabetize the dep list.